.github/workflows/test.yml

```
@@ -7,7 +7,7 @@ on:
 
 jobs:
   test:
-    name: CI - Run Tests and Upload Coverage
+    name: CI - Run Tests and Upload Coverage 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)